common/djangoapps/cme_registration/views.py

```diff
@@ -236,15 +231,9 @@ def _do_cme_create_account(post_vars):
     cme_user_profile.physician_status = post_vars.get('physician_status')
     cme_user_profile.patient_population = post_vars.get('patient_population')
 
```

Comment:
Since we no longer have the `if` conditional stanza here, let's do away with these (now) superfluous newline breaks.

```python
cme_user_profile.patient_population = post_vars.get('patient_population')
cme_user_profile.specialty = post_vars.get('specialty')
cme_user_profile.sub_specialty = post_vars.get('sub_specialty')
cme_user_profile.affiliation = post_vars.get('affiliation')
```